libraries/adaptive-expressions/adaptive/expressions/expression_functions.py

```diff
@@ -14,6 +16,8 @@ def get_standard_functions() -> dict:
     functions.append(Subtract())
     functions.append(Multiply())
     functions.append(Divide())
+    functions.append(Equal())
+    functions.append(LessThen())
```

Comment:
Maybe we can add these to new lines

Reply:
like 
// Math
functions.append(Add())
functions.append(Subtract())
functions.append(Multiply())
functions.append(Divide())

// Comparisons
functions.append(Equal())
functions.append(LessThen())

---
In reply to: [459836534](https://github.com/microsoft/botbuilder-python/pull/1275#discussion_r459836534) [](ancestors = 459836534)

Reply:
fixed